generator/php/resources/templates/README.mustache

```diff
@@ -84,10 +84,6 @@ Class | Method | HTTP request | Description
 
 ## Documentation For Authorization
 
-In order to ease usage of this client library, you might want to use the TokenAutoRefreshClient
-as explained in the [example](examples/).
-
-
 {{^authMethods}}
```

Comment:
To whoever will review this PR: perhaps those next few lines should be removed as well, I'm not too sure, so please review with a critical mind :)